test/request.test.ts

```diff
@@ -1,9 +1,13 @@
+if (parseInt(process.versions.node.split(".")[0]) < 18) {
+  const fetch = require("node-fetch");
+  globalThis.fetch = fetch;
+  globalThis.Headers = fetch.Headers;
+}
```

Comment:
I think we can drop this, as we will stop testing in Nod 16 and below as per https://github.com/octokit/request.js/pull/586

Reply:
Removed in [7b2709e](https://github.com/octokit/request.js/pull/580/commits/7b2709e7fab54b470731e0985bc0158497821950)